packages/rum-core/src/domain/rumEventsCollection/resource/resourceCollection.spec.ts

```diff
@@ -8,14 +8,19 @@ import { RumEventType } from '../../../rawRumEvent.types'
 import { LifeCycleEventType } from '../../lifeCycle'
 import type { RequestCompleteEvent } from '../../requestCollection'
 import { TraceIdentifier } from '../../tracing/tracer'
+import type { RumConfiguration } from '../../configuration'
+import { validateAndBuildRumConfiguration } from '../../configuration'
 import { startResourceCollection } from './resourceCollection'
 
 describe('resourceCollection', () => {
   let setupBuilder: TestSetupBuilder
 
   beforeEach(() => {
     setupBuilder = setup().beforeBuild(({ lifeCycle }) => {
-      startResourceCollection(lifeCycle)
+      startResourceCollection(
+        lifeCycle,
+        validateAndBuildRumConfiguration({ clientToken: 'xxx', applicationId: 'xxx' }) as RumConfiguration
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏using the `!` operator would be more explicit on what we want to do here
```suggestion
        validateAndBuildRumConfiguration({ clientToken: 'xxx', applicationId: 'xxx' })!
```